versions-model-core-extensions/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.codehaus.mojo.versions</groupId>
+    <artifactId>versions</artifactId>
+    <version>2.15.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>versions-model-core-extensions</artifactId>
+  <name>Versions Model Core Extensions</name>
+  <description>Model classes and parser for Core Extensions</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-utils</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.modello</groupId>
+        <artifactId>modello-maven-plugin</artifactId>
+        <configuration>
+          <models>
+            <model>src/main/mdo/core-extensions.mdo</model>
+          </models>
+          <version>1.1.0</version>
+        </configuration>
+        <executions>
+          <execution>
+            <id>generate-java-classes</id>
+            <goals>
+              <goal>xpp3-reader</goal>
+              <goal>java</goal>
+            </goals>
+            <phase>generate-sources</phase>
+          </execution>
+          <execution>
+            <id>generate-site-doc</id>
+            <goals>
+              <goal>xdoc</goal>
```

Comment:
We should not generate xsd for this model - it is internally used for reading extensions.xml